test/evergreen/tsan_warnings.supp

```diff
@@ -12,3 +12,17 @@ race:__evict_update_work
 
 # Stats are allowed to be fuzzy. Ignore races in TSan
 race:src/include/stat.h
+
+# FIXME-WT-12610 Fields like log->slot_state are composite variables and more complex than the usual storing 
+# and loading of ints. These fields will be investigated in a separate ticket. 
+race:src/log
+
+# The ref track code is diagnostic only and allows for races
+race:__ref_save_state
+
+# FIXME-WT-12611 CONNECTION_API_CALL contains a data race that touches the session array, but the number of macro 
+# calls makes diagnosing this very hard. For now suppress the surrounding function and handle this in a separate ticket.
+race:__conn_open_session
+
+# __tiered_queue_peek_empty is intentionally unsafe to avoid taking the tiered lock when it's not needed.
+race:__tiered_queue_peek_empty
```

Comment:
dank whitespace error.

Reply:
Fixed in [75131f6](https://github.com/wiredtiger/wiredtiger/pull/10328/commits/75131f6dae1fc8638481b6ee4484940e878f144e)